packages/cli/src/commands/setup/jobs/jobsHandler.js

```diff
@@ -0,0 +1,140 @@
+}
+`
+
+const getModelNames = async () => {
+  const schema = await getDMMF({ datamodelPath: getPaths().api.dbSchema })
+
+  return schema.datamodel.models.map((model) => model.name)
+}
+
+// TODO(jgmw): This won't handle prisma with schema folder preview feature
+const addDatabaseModel = () => {
+  const schema = fs.readFileSync(getPaths().api.dbSchema, 'utf-8')
+
+  const schemaWithUser = schema + MODEL_SCHEMA
+
+  fs.writeFileSync(getPaths().api.dbSchema, schemaWithUser)
+}
+
+const tasks = async ({ force }) => {
+  const modelExists = (await getModelNames()).includes('BackgroundJob')
+
+  const redwoodVersion =
+    require(path.join(getPaths().base, 'package.json')).devDependencies[
+      '@redwoodjs/core'
+    ] ?? 'latest'
+  const jobsPackage = `@redwoodjs/jobs@${redwoodVersion}`
+
+  return new Listr(
+    [
+      {
+        title: 'Creating job database model...',
+        task: () => {
+          addDatabaseModel()
+        },
+        skip: () => {
+          if (modelExists) {
+            return 'BackgroundJob model exists, skipping'
+          }
+        },
+      },
+      {
+        title: 'Creating config file in api/src/lib...',
+        task: async () => {
+          const isTs = isTypeScriptProject()
+          const outputExtension = isTs ? 'ts' : 'js'
+          const outputPath = path.join(
+            getPaths().api.lib,
+            `jobs.${outputExtension}`,
+          )
+          let template = fs
+            .readFileSync(
+              path.resolve(__dirname, 'templates', 'jobs.ts.template'),
+            )
+            .toString()
+
+          if (!isTs) {
+            template = await transformTSToJS(outputPath, template)
+          }
+
+          writeFile(outputPath, template, {
+            overwriteExisting: force,
+          })
+        },
+      },
+      {
+        title: 'Creating jobs dir at api/src/jobs...',
+        task: () => {
+          try {
+            fs.mkdirSync(getPaths().api.jobs)
+          } catch (e) {
+            // ignore directory already existing
+            if (!e.message.match('file already exists')) {
+              throw new Error(e)
+            }
+          }
+          writeFile(path.join(getPaths().api.jobs, '.keep'), '', {
+            overwriteExisting: force,
+          })
+        },
+      },
+      addApiPackages([jobsPackage]),
+      {
+        title: 'One more thing...',
+        task: (_ctx, task) => {
+          task.title = `One more thing...
+
+          ${c.success('\nBackground jobs configured!\n')}
+
+          ${!modelExists ? 'Migrate your database to finish setting up jobs:\n' : ''}
+          ${!modelExists ? c.warning('\n\u00A0\u00A0yarn rw prisma migrate dev\n') : ''}
+
+          Generate jobs with: ${c.warning('yarn rw g job <name>')}
+          Execute jobs with:  ${c.warning('yarn rw jobs work\n')}
+
+          Check out the docs for more info:
+          ${c.link('https://docs.redwoodjs.com/docs/background-jobs')}
```

Comment:
This link colour is only just visible on a terminal with a white background.

Reply:
Yeah we really need to update that `link` color for everyone. I like the fact that we have a separate `colors` setting for it (like underlining it, perhaps) but the color choice needs an update! Or just remove the color and only provide the underline.